.NET/Microsoft.Recognizers.Text.Number/Extractors/BaseNumberRangeExtractor.cs

```diff
@@ -143,9 +145,19 @@ private void GetMatchedStartAndLength(Match match, string type, string source, o
             else
             {
                 var numberStr = string.IsNullOrEmpty(numberStr1) ? numberStr2 : numberStr1;
-
+                
                 var extractNumList = ExtractNumberAndOrdinalFromStr(numberStr);
```

Comment:
ExtractNumberAndOrdinalFromStr should not be in the NumberRangeExtractor as it doesn't handle duration concepts. Let's at least at a @TODO in the code to refactor and move it to a better place. Doesn't need to be in this PR.

Reply:
Added.